public/nextjs_migration/client/js/addEmail.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+let dialogEl, form
+
+function init () {
+  dialogEl = document.querySelector('dialog[data-partial="addEmail"]')
+  if (!dialogEl) return
+
+  form = dialogEl.querySelector('form')
+  form.addEventListener('submit', handleSubmit)
+  dialogEl.addEventListener('close', kill)
+}
+
+async function handleSubmit (e) {
+  e.preventDefault()
+
+  try {
+    form.elements['email-submit'].toggleAttribute('disabled', true)
+    const res = await fetch('/api/v1/user/email', {
+      headers: {
+        'Content-Type': 'application/json',
```

Comment:
Ran a quick diff on the previous file: 

Question: Why was [this line](https://github.com/mozilla/blurts-server/blob/main/src/client/js/partials/addEmail.js#L24) removed? 

```
'x-csrf-token': form.elements['csrf-token'].value,
```